spyder_line_profiler/spyder/plugin.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Copyright © 2021, Spyder Line Profiler contributors
+#
+# Licensed under the terms of the MIT license
+# ----------------------------------------------------------------------------
+"""
+Spyder Line Profiler 5 Plugin.
+"""
+
+# Third-party imports
+from qtpy.QtGui import QIcon
+
+# Spyder imports
+from spyder.api.plugins import Plugins, SpyderDockablePlugin
+from spyder.api.translations import get_translation
+from spyder.api.plugin_registration.decorators import on_plugin_available
+from spyder.api.plugin_registration.decorators import on_plugin_teardown
+from qtpy.QtCore import Signal
+
+# Local imports
+from spyder.plugins.mainmenu.api import ApplicationMenus
+from spyder_line_profiler.spyder.confpage import SpyderLineProfilerConfigPage
+from spyder_line_profiler.spyder.widgets import SpyderLineProfilerWidget
+from spyder_line_profiler.spyder.widgets import SpyderLineProfilerWidgetActions
+from spyder_line_profiler.spyder.widgets import is_lineprofiler_installed
+
+_ = get_translation("spyder_line_profiler.spyder")
+
+
+
+class SpyderLineProfilerActions:
+    # Triggers
+    Run = 'run_action_menu'
```

Comment:
```suggestion
    Run = 'run_profiler_action'
```

I think this is more telling.